rasa_core/train.py

```diff
@@ -189,6 +224,54 @@ def train_dialogue_model(domain_file, stories_file, output_path,
     return agent
 
 
+def train_comparison_models(story_filename,
+                            domain,
+                            epochs,
+                            output_path=None,
+                            exclusion_percentage=None,
+                            starspace=True,
+                            max_history=None):
+
+    """Trains either a KerasPolicy model or an EmbeddingPolicy, excluding a
+    certain percentage of a story file"""
+
+    if starspace:
```

Comment:
instead of hardcoding this here, can we maybe use this PR to define different models in different files: https://github.com/RasaHQ/rasa_core/pull/1169

Reply:
so you mean pass as an argument different model configs?

Reply:
could be an option: so instead of only passing in one policy.yml (or whatever we are going to call this) specify multiple and compare these

Reply:
ok yeah that's a good idea :)